borg/archiver.py

```diff
@@ -1353,6 +1394,13 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('--keep-tag-files', dest='keep_tag_files',
                                action='store_true', default=False,
                                help='keep tag files of excluded caches/directories')
+        subparser.add_argument('--pattern', dest='pattern',
+                               type=parse_inclexcl_pattern, action='append',
+                               metavar="PATTERN", help='include/exclude paths matching PATTERN')
+        subparser.set_defaults(pattern=[])
```

Comment:
Would it make sense to move this default (+line 1482) to the top-level default declaration in line 1137?

(Nitpick: Shouldn't this variable be named `patterns` (plural)?)